src/App/integrationTests/App.addSampleUnitRouting.test.js

```diff
@@ -110,23 +119,26 @@ test('Clicking Add Sample Unit then click Habitat Complexity link expects to see
 test('Clicking Add Sample Unit then click Benthic LIT link expects to see New Benthic LIT page.', async () => {
   const { dexiePerUserDataInstance, dexieCurrentUserInstance } = getMockDexieInstancesAllSuccess()
 
-  renderAuthenticatedOnline(<App dexieCurrentUserInstance={dexieCurrentUserInstance} />, {
-    initialEntries: ['/projects/5/collecting'],
-    dexiePerUserDataInstance,
-    dexieCurrentUserInstance,
-  })
+  const { user } = renderAuthenticatedOnline(
```

Comment:
The wrapper utilities will run `userEvent.setup()` for you and export the returned user for use in tests